### ISSUE TITLE
Cmd arguments - sensor size

### ISSUE THREAD

**Issue by simonbjork** [stuffmatic/fSpy#50]
It would be great to be able to send filmback info to fspy as a command line argument when launching. Something like:

fspy.exe -sensor_size 26.4x14.85

It would be useful for an exporter tool.

**Comment by stuffmatic** [comment]
Could you describe the workflow you have in mind with a bit more detail?

**Comment by simonbjork** [comment]
Yes, when working in a prodoction it's often possible to query the camera filmback from a database. So if I were to launch fspy from a separate app that knows the sensor size, I wouldn't have to change those values each time.

**Comment by stuffmatic** [comment]
This only applies to single vanishing point calibration, right? Because when two vanishing points are used, fSpy doesn't need to know the dimensions of the sensor.

**Comment by simonbjork** [comment]
Ah, you're right. But when I import a camera, I want it to use the correct sensor size etc. But I guess that could be calculated on the importer side. Still, it's nice to see the focal lenght estimation in the GUI so you know if you're somewhat correct in the match.
